crates/holochain/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+- `device_seed_lair_tag` is now part of `ConductorConfig`. This was previously a field as part of the optional DPKI config. Now, a device seed should be specified even if not using DPKI. If the device seed is specified, when installing an app without providing an agent key, a new agent key will be generated by deriving a key from the device seed using the total number of apps ever installed as part of the derivation path. If the device seed is not specified, it will not be possible to install an app without specifying an agent key (app installation will error out).
+- `allow_throwaway_random_agent_key` can be set in the `InstallAppPayload` to override the aforementioned behavior, allowing an agent key to not be specified even if a device seed is not specified in the conductor. This is a safety mechanism, and should only be used in test situations where the generated agent key is a throwaway and will never need to be recovered.
+- Holochain now actually makes use of the device seed, which was previously ignored
+- Holochain now makes sure to properly register in DPKI any pregenerated agent key which is provided in app installation, when DPKI is enabled.
```

Comment:
missing pr numbers?

Reply:
I haven't been adding PR numbers for a while, and at least one other of us hasn't either. I can start again. It seems easy enough to `git log x..y` where x and y are subsequent release tags and surmise which squashed commit corresponds to the PR that introduced the change you're interested in. Then again adding a PR number is minimal effort :)

Reply:
I assumed it was more about app devs being able to click through the changelog to get additional details.

Reply:
Wouldn't help with this one, the PR details say "read the docs and the changelog"